ChangeLog

```diff
@@ -13,6 +13,8 @@ Default behavior changes
      Applied Sciences).
 
 Security
+   * Fix buffer overread vulnerability in ssl_parse_server_psk_hint
+     function.
```

Comment:
The changelog should have a bit more information than this. It doesn't need an extensive vulnerability analysis, but it needs to summarize how dangerous the issue is. For example, by approximately how many bytes is the overread? Is it likely to cause an information leak or a crash? See previous examples of changelog entries for inspiration.